issue_07/script.js

```diff
@@ -0,0 +1,49 @@
+function startLoading() {
+    const div = document.getElementById('js-loading');
```

Comment:
The name of this variable is a div, which is too common, so it is better to use a specific name.
For example, "loadingPlace".

Reply:
The variable name you suggested is easier to understand what the variable is for. 
Fixed.
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/eb050e9ba93c40e5af62e551a4af80a5e3356ec3